server/app.js

```diff
@@ -1,70 +1,29 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const path = require('path');
-const db = require('../db');
+const db = require('../db/index.js');
 
 const app = express();
 
-
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use('/books/:id', express.static(path.join(__dirname, '/../public')));
-
-// get and post routes to interact with database here
 
-// get all reviews for specific book id
-app.get('/books/:id/reviews', async (req, res) => {
-  const { id } = req.params;
-  try {
-    const reviews = await db.getReviews(id);
-    res.json(reviews);
-  } catch (err) {
-    res.json(err);
-  }
+app.get('/', (request, response) => {
+  response.json({ info: 'Node.js, Express, and Postgres API' });
```

Comment:
is this endpoint necessary?

Reply:
nope. That was just there let me know I was connected when testing the server in Postman.
Deleted.